Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs

```diff
@@ -63,7 +79,7 @@ internal static class ImageViewExtensions
 				return;
 			}
 
-			if (!imageView.IsDisposed())
+			if (!setByImageViewHandler && !imageView.IsDisposed())
```

Comment:
We should probably reinstate this code

https://github.com/xamarin/Xamarin.Forms/blob/e8660383b03b4010724b36c6eef471a2e74e7c98/Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs#L55

Because AFAICT anyone that followed your advice on the blog
http://jonathanpeppers.com/Blog/xamarin-forms-performance-on-android

Will end up with broken images after this PR

Reply:
I had that thought initially as well but I'm pretty sure the export of the handler causes the default one to just get totally replaced 

when I added in 
```C#
[assembly: ExportImageSourceHandler(typeof(FileImageSource), typeof(xfperf.FileImageSourceHandler))]

namespace xfperf
{
    public class FileImageSourceHandler : IImageSourceHandler
    {
        public Task<Bitmap> LoadImageAsync(ImageSource imagesource, Context context, CancellationToken cancelationToken = default(CancellationToken))
        {
            return Task.FromResult<Bitmap>(null);
        }
    }
}
```

It caused this line
```C#
var imageViewHandler = Internals.Registrar.Registered.GetHandlerForObject<IImageViewHandler>(source);
``` 

to return null and fall through to the else statement part and then call into the xfperf handler

Reply:
I put it back